### PR TITLE
Feature/1/database

### PR DESCRIPTION
**Pull request by amelie-certin**

### CODE REVIEW

File: app/models/classrooms_user.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ClassroomsUser < ActiveRecord::Base
+  # Relations
+  belongs_to :classroom
+  belongs_to :user
+
```
Comment (alex-lairan) [inline app/models/classrooms_user.rb:7]: Presence of `as`

File: app/models/classroom.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  has_many :users, through: :classrooms_users
+  has_many :homework
+  has_and_belongs_to_many :subjects
+
+  # Validations
+  validates :name, presence: true
+  validates :name, uniqueness: {
+    scope: :school,
+    message: 'This class already exits for this school.'
+  }, unless: proc { |classroom| classroom.school_id.nil? }
```
Comment (alex-lairan) [inline app/models/classroom.rb:16]: It could be `if: :classroom` no?

File: app/models/homeworks_user.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class HomeworksUser < ActiveRecord::Base
+  # Relations
+  belongs_to :user
+  belongs_to :homework
+
```
Comment (alex-lairan) [inline app/models/homeworks_user.rb:7]: Presence of `progress`

**Review by alex-lairan — CHANGES_REQUESTED** [review]
Rebase, you have devise commit

**Comment by amelie-certin** [comment]
Cette branche est encore utile ?

**Comment by alex-lairan** [comment]
Je ne pense pas

**Pull request closed without merging**